zkvm/src/verifier.rs

```diff
@@ -67,6 +67,10 @@ impl<'a, 'b> Delegate<r1cs::Verifier<'a, 'b>> for Verifier<'a, 'b> {
         Ok(Some(instr))
     }
 
+    fn new_run(&self, prog: Data) -> Result<Self::RunType, VMError> {
+        Ok(VerifierRun::new(prog.to_bytes().to_vec()))
```

Comment:
`data.to_bytes()` already gives a `Vec<u8>`, no?

```suggestion
        Ok(VerifierRun::new(prog.to_bytes()))
```